db/warehouse/migrate/20210702144442_set_date_added_for_cohort_clients.rb

```diff
@@ -0,0 +1,9 @@
+class SetDateAddedForCohortClients < ActiveRecord::Migration[5.2]
+  def up
+    PaperTrail.request(enabled: false) do # Migration can't find 'versions' table?
+      GrdaWarehouse::CohortClient.find_each do |cohort_client|
+        cohort_client.update(date_added_to_cohort: cohort_client.created_at.to_date)
+      end
+    end
+  end
```

Comment:
This shouldn't assume the current cohort client has the correct date.  It should look back at `GrdaWarehouse::CohortClientChange` to find the most recent add.

Reply:
I think this is addressed below...